sycl/test-e2e/Basic/kernel_max_wg_size.cpp

```diff
@@ -5,6 +5,7 @@
 // https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_intel_grf_size.asciidoc
 // REQUIRES: arch-intel_gpu_pvc || gpu-intel-dg2
 // UNSUPPORTED: cuda || hip
+// UNSUPPORTED-TRACKER: INTENDED - This extension is currently implemented in DPC++ only for Intel GPU devices and only when using the Level Zero backend or OpenCL backend.
```

Comment:
If we're going to call it `UNSUPPORTED-TRACKER`, this sort of case seems weird. Could we then name it `UNSUPPORTED-INTENDED` for these cases and a case of `UNSUPPORTED` must have one or the other following it? I.e.
```suggestion
// UNSUPPORTED-INTENDED: This extension is currently implemented in DPC++ only for Intel GPU devices and only when using the Level Zero backend or OpenCL backend.
```

Reply:
That'd better. 7974dbf